goship.go

```diff
@@ -321,6 +331,7 @@ func DeployOutputHandler(w http.ResponseWriter, r *http.Request, env string, for
 }
 
 func DeployLogHandler(w http.ResponseWriter, r *http.Request, full_env string, environment goship.Environment) {
+	u, _ := getUser(r)
```

Comment:
I don't see it being logged in the `getUser` function, but wouldn't recommend logging there anyway - logging is usually best left to the discretion of the caller, based on the kind of error returned. I think it's important to distinguish between the case where we expect a user but don't get one, and where we don't expect a user. 

I suggest checking the error here to see if it's a serious one (did we fail to load from the session?) or a non-serious one (maybe this user has no avatar set), and then decide whether want to use the default user returned by the function, or log an error and return.